software/py/bsg_manycore_link_gen.py

```diff
@@ -235,7 +235,7 @@ def script(self):
       # .text section virtual address starts at 0x0 but
       # loaded at 0x80000000
       if sec == ".text.dram":
-        vaddr = "0x0"
+        vaddr = "0x80000000"
```

Comment:
should be parameterized based on glen

Reply:
with a comment about PC width requirements

Reply:
This is changed in a new commit